db/migrations/20230327080656_create_users_table.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  kakao_id BIGINT NOT NULL,
+  email VARCHAR(100) ,
+  password VARCHAR(200),
```

Comment:
- 카카오로 회원가입을 진행할 경우 비밀번호를 받지 않기 때문에 password에 null을 주어야 합니다.
- 반대로 일반 회원가입도 언제가는 진행할 수 있기 때문에 kakao_id에도 null을 주면 좋을 것 같습니다.
```suggestion
  kakao_id BIGINT NULL,
  email VARCHAR(100) ,
  password VARCHAR(200) NULL,
```